spec/ddtrace/contrib/active_record/integration_spec.rb

```diff
@@ -56,6 +56,26 @@
     end
   end
 
+  describe '#auto_instrument?' do
+    subject(:auto_instrument?) { integration.auto_instrument? }
+
+    context 'outside of a rails application' do
+      before do
+        allow(Datadog::Utils::Rails).to receive(:railtie_supported?).and_return(false)
+      end
+
+      it { is_expected.to be(true) }
+    end
+
+    context 'when within a rails application' do
+      before do
+        allow(Datadog::Utils::Rails).to receive(:railtie_supported?).and_return(true)
+      end
+
+      it { is_expected.to be(false) }
+    end
+  end
```

Comment:
I suggested you extract this into a shared example, as it seems like we use it with the exact same code in many Rails sub-gems.

Reply:
makes sense, updated